analyzers/src/SonarAnalyzer.Common/Facade/SyntaxFacade.cs

```diff
@@ -33,6 +33,7 @@ public abstract class SyntaxFacade<TSyntaxKind>
     public abstract bool IsAnyKind(SyntaxNode node, params TSyntaxKind[] syntaxKinds);
     public abstract bool IsAnyKind(SyntaxTrivia trivia, params TSyntaxKind[] syntaxKinds);
     public abstract bool IsKnownAttributeType(SemanticModel model, SyntaxNode attribute, KnownType knownType);
+    public abstract bool AreEquivalent(SyntaxNode oldNode, SyntaxNode newNode);
```

Comment:
Why oldNode and newNode? Seems misleading to me.

Reply:
This is how Microsoft.CodeAnalysis is naming those parameters, 

```cs
public static bool AreEquivalent(SyntaxNode oldNode, SyntaxNode newNode, Func<SyntaxKind, bool> ignoreChildNode = null)
        {
            return SyntaxEquivalence.AreEquivalent(oldNode, newNode, ignoreChildNode, topLevel: false);
        }
```

Reply:
Yes. I know. But that is because the API is in SyntaxFactory. There it makes sense, but in our case not so much. There is e.g. https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.syntaxnode.isequivalentto (Roslyn > 3.0). There it is named `other`.